roles/pulp_devel/tasks/install_podman.yml

```diff
@@ -45,20 +45,37 @@
   ignore_errors: '{{ ansible_facts.distribution_version == "8.1" }}'
 
 - name: Adding podman registries (Debian-specific)
-  command: echo -e "[registries.search]\nregistries = ['docker.io', 'quay.io']" | tee /etc/containers/registries.conf
+  # This should have been a toml_file module if I had found one
+  ini_file:
+    path: /etc/containers/registries.conf
+    section: registries.search
+    option: registries
+    value: "['docker.io', 'quay.io']"
   become: yes
-  args:
-    creates: /etc/containers
   when: ansible_facts.distribution == 'Debian'
 
+- name: Adding podman insecure registries (Debian-specific)
+  # This should have been a toml_file module if I had found one
```

Comment:
Probably fedora boxes want this, too, but the file is in a different format IIRC.